_search-plugins/concurrent-segment-search.md

```diff
@@ -117,8 +192,20 @@ Non-concurrent search calculates the document count error and returns it in the
 For more information about how `shard_size` can affect both `doc_count_error_upper_bound` and collected buckets, see [this GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/11680#issuecomment-1885882985).
 
 
-## Developer information: AggregatorFactory changes
+## Developer information
+
+The following sections provide additional information for developers.
+
+### AggregatorFactory changes
+
+Because of implementation details, not all aggregator types can support concurrent segment search. To accommodate this, we have introduced a [`supportsConcurrentSegmentSearch()`](https://github.com/opensearch-project/OpenSearch/blob/2.x/server/src/main/java/org/opensearch/search/aggregations/AggregatorFactory.java#L123) method in the `AggregatorFactory` class to indicate whether a given aggregation type supports concurrent segment search. By default, this method returns `false`. Any aggregator that needs to support concurrent segment search must override this method in its own factory implementation. 
```

Comment:
"Because of the implementation configuration"?

Reply:
This is actually existing wording: https://opensearch.org/docs/latest/search-plugins/concurrent-segment-search/#developer-information-aggregatorfactory-changes

Reply:
IMO we probably don't need to make changes to this paragraph since it's already published, I think it's just showing up in this PR because it was moved.

Reply:
@jed326 I think "implementation details" reads better because it's commonly used.